tracker/view/login.py

```diff
@@ -86,7 +86,7 @@ def logout():
 def sso_auth():
     try:
         token = oauth.idp.authorize_access_token()
-        parsed_token = oauth.idp.parse_id_token(token)
+        parsed_token = token['userinfo']
```

Comment:
This is not automatically given and can lead to a 500 KeyError, which i'd like to avoid. The `userinfo` key in the token dict will only be available in via `authorize_access_token()` when:
- `id_token` in token and `nonce` in state_data

Whenever that is not given, we will get a token dict without `userinfo` hence we will need to guard the code against the lack of the key and raise a bad request with a new error identifier.